jekyll/_cci2/configuration-reference.md

```diff
@@ -163,6 +164,22 @@ jobs:
     machine: true
 ```
 
+CircleCI supports multiple machine images that can be specified in `image` field:
+
+* `circleci/classic:latest` (default) - an Ubuntu Trusty-based image with Docker `17.03.0-ce` along with common language tools found in CircleCI 1.0 build image.  The `latest` channel provides the latest tested images, changes to the channel are announced at least a week in advance.
+* `circleci/classic:edge` - an Ubuntu Trusty-based image with Docker `17.06.0-ce` along with common language tools found in CircleCI 1.0 build image.  The `edge` channel provides release candidates that will eventually.
+
+So you can set the following to use Docker `17.06.0-ce`:
+
+```YAML
+jobs:
+  build:
+    machine:
+      image: circleci/classic:edge
+```
+
+*Coming soon*: Capability to pin the image version and documentation of the actual content of each released image.
```

Comment:
I would rather not have this 'coming soon' information for customers, but it can be a comment in the file for us to remember to do later.